client/src/components/songDetails/songDetails.css

```diff
@@ -0,0 +1,45 @@
+
+  @import url('https://fonts.googleapis.com/css2?family=ABeeZee:ital@0;1&display=swap');
+
```

Comment:
did you add mobile first? media queries? remember that this design works only for mobile screens - not tablet/laptop/pc